tests/test_views.py

```diff
@@ -388,7 +388,20 @@ def test_with_empty_get_tables_list(self):
         class View(tables.MultiTableMixin, TemplateView):
             template_name = "multiple.html"
 
-            def get_tables(self):
+            def get_tables(self, **kwargs):
+                return []
+
+        response = View.as_view()(build_request("/"))
+        response.render()
+
+        html = response.rendered_content
+        self.assertIn("<h1>Multiple tables using MultiTableMixin</h1>", html)
+
+    def test_with_empty_get_tables_clases_list(self):
+        class View(tables.MultiTableMixin, TemplateView):
+            template_name = "multiple.html"
+
+            def get_tables_classes(self):
```

Comment:
This test is implicit in checking the `get_tables_classes` method is called. I think it would be an improvement if we somehow make it explicit, either by returning something we check, or checking if the method is indeed called.